packages/go_router/lib/src/configuration.dart

```diff
@@ -48,7 +48,7 @@ class RouteConfiguration {
   String namedLocation(
     String name, {
     Map<String, String> params = const <String, String>{},
-    Map<String, String> queryParams = const <String, String>{},
```

Comment:
I'm concerned that it's not clear how to use this `dynamic`. Would a breaking change be better instead, especially since we're preparing for v5? 

Potential types we could accept:

1. `Map<String, List<String>>`, or...
2. ... `List<QueryParameter>`: where `QueryParameter` is a tuple of two strings for the key/value

Reply:
Ah never mind, this follows what `Uri`'s constructor does: https://api.dart.dev/stable/2.18.0/dart-core/Uri/Uri.html